modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -1004,6 +1007,19 @@ private void _deleteObjectFields(
 		}
 	}
 
+	private void _deleteObjectRelationshipByParameterFieldId(
+			long parameterFieldId)
+		throws PortalException {
+
+		for (ObjectRelationship objectRelationship :
+				objectRelationshipPersistence.findByParameterObjectFieldId(
+					parameterFieldId)) {
+
+			objectRelationshipLocalService.deleteObjectRelationship(
+				objectRelationship);
+		}
```

Comment:
Hey @leilanyaragao,
Since this is only used once, I think we can inline this logic for now.
Could you put this logic (FOR) before this line `_objectFieldLocalService.deleteRelationshipTypeObjectField()` please?

Reply:
Hey @marcelabc 

My first approach was inline. But in this case, we will need to use `objectRelationship1` in the FOR. Because of this, I preferred to create a new private method.

Reply:
Sorry, I don't understand. 
I don't see this variable `objectRelationship1` in the private method. Could explain it please?
We can talk in Slack.